test/integration/etcd/data.go

```diff
@@ -314,6 +314,22 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		},
 		// --
 
+		// k8s.io/kubernetes/pkg/apis/flowcontrol/v1beta3
```

Comment:
the tests are currently disabled
https://github.com/kubernetes/kubernetes/blob/d5e5f48b8805dcbd321675249bfba89fe997ff99/test/integration/apiserver/apply/reset_fields_test.go#L75-L79

we need https://github.com/kubernetes/kubernetes/pull/110173 to merge.